ICAFIX/ReApplyFixMultiRunPipeline.sh

```diff
@@ -493,16 +510,18 @@ if (( regenConcatHP )); then
 				# (Necessary to protect the semicolons that separate matlab commands, which would otherwise
 				# get interpreted as separating different bash shell commands)
 				# See note below about why we export FSL_FIX_WBC after sourcing FSL_FIXDIR/settings.sh
-				(
-					#fix's default settings.sh isn't safe to unbound variables or exit codes, so disable everything before sourcing
-					debug_disable_trap
-					set +u
-					source "${FSL_FIXDIR}/settings.sh"
-					set -u
-					debug_enable_trap
-					export FSL_FIX_WBC="${Caret7_Command}"
-					"${interpreter[@]}" <<<"${matlab_code}"
-				)
+				if ((legacyFix));then
+					(
+						#fix's default settings.sh isn't safe to unbound variables or exit codes, so disable everything before sourcing
+						debug_disable_trap
+						set +u
+						source "${FSL_FIXDIR}/settings.sh"
+						set -u
+						debug_enable_trap
+						export FSL_FIX_WBC="${Caret7_Command}"
+						"${interpreter[@]}" <<<"${matlab_code}"
+					)
+				fi
```

Comment:
Why isn't there an `else` here?  If there is a substantial intentional control flow difference between using legacy or pyfix, please add comments explaining it.